### ISSUE TITLE
Support passing params as JSON object

### ISSUE THREAD

**Issue by zhaoyao91** [ice-services/moleculer-repl#5]
This package use vorpal options as action params. However, the expression ability of vorpal options is not enough. For example, I can't pass in array of objects.

How about support call with json? for example:

```
// Register broker.call json
  vorpal
    .command("callWithJson <actionName> <jsonPayload>", "Call an action")
    .autocomplete({
      data() {
        return _.uniq(broker.registry.getActionList({}).map(item => item.action.name));
      }
    })
    .allowUnknownOptions()
    .action((args, done) => {
    	console.log(args.jsonPayload)
      const payload = JSON.parse(args.jsonPayload)
      console.log(chalk.yellow.bold(`>> Call '${args.actionName}' with params:`), payload);
      broker.call(args.actionName, payload)
        .then(res => {
          console.log(chalk.yellow.bold(">> Response:"));
          console.log(util.inspect(res, { showHidden: false, depth: 4, colors: true }));
        })
        .catch(err => {
          console.error(chalk.red.bold(">> ERROR:", err.message));
          console.error(chalk.red.bold(err.stack));
          console.error("Data: ", util.inspect(err.data, { showHidden: false, depth: 4, colors: true }));
        })
        .finally(done);
    });
```

**Comment by icebob** [comment]
I don't want to duplicate the `call` command code. I would rather add a `-j`, `--json` option to `call` and `dcall` commands. In this case, it will parse the params as a JSON string.

**Comment by icebob** [comment]
Example:
```sh
mol $ call greeter.hello --json '{"name": "John"}'
```

**Comment by zhaoyao91** [comment]
well, that is fine. look forward to it :)

**Comment by icebob** [comment]
I changed the implementation. No need `--json` option.
New logic:
```bash
# Params as options
mol $ call greeter.hello --name John

# Params as JSON string
mol $ call greeter.hello '{"name": "John"}'
```

Released in v0.3.2
